packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -4,14 +4,34 @@ import {
   ParseAnnotation,
   is
 } from "@atjson/document";
-import OffsetSource, { LineBreak, Paragraph } from "@atjson/offset-annotations";
+import OffsetSource, {
+  LineBreak,
+  Paragraph,
+  ListItem,
+  List
+} from "@atjson/offset-annotations";
 import { Heading } from "./annotations";
 import GDocsSource from "./source";
 
 // eslint-disable-next-line no-control-regex
 const VERTICAL_TABS = /\u000B/g;
 const NEWLINE_PARAGRAPH_SEPARATOR = /\n(\s*\n)*/g;
 
+function compareAnnotations(a: Annotation<any>, b: Annotation<any>) {
```

Comment:
I think we have a ticket to make this the default (if this is what we want our stable sort to be). If not, I'd love to capture this and maybe remove it.

Reply:
Yep, I imagine whenever we pick up that ticket we will remove these cases where we override the default behavior. But changing the default sorting behavior is a breaking change and I'd rather not couple that with this.